src/utilities/schemaPrinter.js

```diff
@@ -194,9 +194,13 @@ function printObject(type: GraphQLObjectType, options): string {
 }
 
 function printInterface(type: GraphQLInterfaceType, options): string {
+  const interfaces = type.getInterfaces();
```

Comment:
please extract it into `printImplementedInterfaces` and reuse for both objects and interfaces, similar to `printFields`

Reply:
👍